packages/template-retail-react-app/app/pages/checkout/index.jsx

```diff
@@ -62,11 +62,6 @@ const Checkout = () => {
         setIsLoading(true)
         try {
             const order = await createOrder({
-                // We send the SLAS usid via this header. This is required by ECOM to map
-                // Einstein events sent via the API with the finishOrder event fired by ECOM
-                // when an Order transitions from Created to New status.
-                // Without this, various order conversion metrics will not appear on reports and dashboards
-                headers: {_sfdc_customer_id: usid},
```

Comment:
We'll need to do the same thing to the older PWA Kit: https://github.com/SalesforceCommerceCloud/pwa-kit/blob/release-2.6.x/packages/template-retail-react-app/app/commerce-api/hooks/useBasket.js#L367-L374

Reply:
No net new customer should use v2 and existing customers will need to patch their project, therefore i don't think we need to patch v2 pwa kit?

Reply:
I see 🤔 good point, all right we don't need to patch v2.